docs/kotlin/getting-started.md

```diff
@@ -62,7 +62,7 @@ $ cd proto
 Next, scaffold a basic [`buf.yaml`][buf.yaml] by running `buf mod init`.
 
 ```bash
-$ buf mod init
+$ buf config init
```

Comment:
We need to update above on line 62 as well to replace `mod init` with `config init`.